src/libponyc/CMakeLists.txt

```diff
@@ -120,7 +120,22 @@ endif (NOT MSVC)
 
 # build a standalone version of libponyc.a with all needed dependencies linked statically
 if (MSVC)
-    # TODO
+    # this will not include LLVM-C.lib as it contains lots of duplicated definitions
+    file(GLOB_RECURSE LLVM_OBJS "${PROJECT_SOURCE_DIR}/../../build/libs/lib/LLVM[!-]*.lib")
+
+    set(libponyc_standalone_file "${CMAKE_STATIC_LIBRARY_PREFIX}ponyc-standalone${CMAKE_STATIC_LIBRARY_SUFFIX}")
+    set(libponyc_standalone_path "${CMAKE_BINARY_DIR}/${libponyc_standalone_file}")
+    # TODO: do we need to get any static libstc++ or other runtime libraries here?
```

Comment:
what's up with this TODO?

Reply:
As it is working as is, this is TODO can be deleted.